src/reducers/index.js

```diff
@@ -1,48 +1,60 @@
-
+// Copyright 2018, 2019 Stanford University see Apache2.txt for license
 import { combineReducers } from 'redux'
 import { createSelector } from 'reselect'
 import { generateLD } from './linkedData'
 import lang from './lang'
 import authenticate from './authenticate'
-import { removeMyItem, setMyItems } from './literal'
-
-const removeAllContent = () => {
+import { removeAllContent, setMyItems, removeMyItem } from './literal'
 
-}
 
-const resourceTemplateSelector = (state, id) => state[id]
+const resourceTemplateSelector = (state, props) => {
 
-const propertySelector = (state, rtId, id) => state[rtId][id]
+  const resTemp = state.selectorReducer[props.rtId]
+  if (resTemp == undefined) {
+    return state
+  }
+  let items
+  if (props.propertyTemplate.propertyURI in resTemp) {
+    items = resTemp[props.propertyTemplate.propertyURI]
+  } else {
+    resTemp[props.propertyTemplate.propertyURI] = []
+    items = []
+  }
+  return items
+}
 
 export const getProperty = createSelector(
-  [resourceTemplateSelector, propertySelector],
-  (resourceTemplate, propertyURI)  => {
-    return resourceTemplate[propertyURI]
+  [ resourceTemplateSelector ],
```

Comment:
I assume you will be discussing exactly how this works next week, so after some smoke testing I will be able to merge this.

Reply:
Cool! Yes, this will be a core part of how this works next week. Thanks for creating tickets [429]( https://github.com/LD4P/sinopia_editor/issues/429) and [430](https://github.com/LD4P/sinopia_editor/issues/430) for follow-up.